src/chai-form-configurator.ts

```diff
@@ -0,0 +1,414 @@
+import { ChaiForm } from './chai-form';
+
+@customElement('chai-form-configurator')
+export class ChaiFormConfigurator extends LitElement {
+  @property({ type: String }) targetForm: string = '';
+
+  static override styles = css`
+   
+    .form-configurator-container
+    {
+      color: black;
+    padding: 20px;
+    background-color: #fbf7f1;
+    border-radius: 20px;
+    text-shadow: none;
+    font-weight: bolder;
+    font-size: 14px;
+    }
+
+    .row
+    {
+      margin-bottom:15px;
+      font-family:sans-serif;
+    }
+    .form-configurator-header-message
+    {
+      margin-bottom: 34px;
+    font-weight: bold;
+    }
+  `;
+
+  constructor() {
+    super();
+    this.targetForm = ''; 
+  }
+
+  @property({ type: String }) selectedVisibilityValue: string = '';
+
+
+  toggleChaiFormSlots(event : Event ,  componentName : string)
+  {
+
+    
+     const target = document.getElementById(this.targetForm) as ChaiForm; 
+        if (!target) return;
+
+            
+
+          const checkbox = event.target as HTMLInputElement; 
+        if(!checkbox) return;
+
+        if(componentName == 'chai-name')
+        {
+          target.isCreateName = checkbox.checked;
+        }else if(componentName == 'chai-email')
+        {
+          target.isCreateEmail = checkbox.checked;
+        }else if(componentName == 'chai-address')
+        {
+          target.isCreateAddress = checkbox.checked;
+        }else if(componentName == 'chai-phone')
+        {
+          target.isCreatePhone = checkbox.checked;
+        }
+  }
+
+
+  applyChaiStyle(event: Event , cssVaraible:string , px:string)
+  {
+    console.log(event,cssVaraible);
+      const target = document.getElementById(this.targetForm) as ChaiForm; 
+        if (!target) return;
+        const input = event.target as HTMLInputElement;
+        target.style.setProperty(cssVaraible , input.value + px);
+        
+  }
+
+
+  override render() {
+    return html`
+        <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/bootstrap@4.5.3/dist/css/bootstrap.min.css" integrity="sha384-TX8t27EcRE3e/ihU7zmQxVncDAy5uIKz4rEkgIXeMed4M0jlfIDPvg6uqKI2xXr2" crossorigin="anonymous">
+
+<div>
+ 
+ <div class="row form-configurator-container">
+  <div class="col-sm-12">
+     <h2 class="form-configurator-header-message">Form Configurator</h2>
+  </div>
+        <div class="col-sm-6">
+            <div class="row">
+                <div class="col-sm-12">
+                    <label for="form-flex-direction">Form Flex Direction</label>
+                    <select id='form-flex-direction' class="form-control" @input="${(event:CustomEvent) => {this.applyChaiStyle(event,'--chai-form-flex-direction','')}}">
+                        <option value=''>Choose Flex Direction</option>
+                        <option value='column'>Column</option>
+                        <option value='row'>row</option>
+                    </select>
+                </div>
+            </div>
```

Comment:
There is _so_ much repetitive code here. Create some reusable components to make this maintainable.